packages/components/src/internal/components/productnavigation/ProductNavigationMenu.tsx

```diff
@@ -97,36 +94,33 @@ export const ProductNavigationMenuImpl: FC<ProductNavigationMenuImplProps> = mem
         return <Alert>{error}</Alert>;
     }
 
-    if (!products || !projects || !tabs) {
+    if (!products || !tabs) {
         return <LoadingSpinner wrapperClassName="product-navigation-loading-item" />;
     }
 
     const selectedProduct = getSelectedProduct(products, selectedProductId);
-    const productProjects = selectedProduct ? productProjectMap[selectedProduct.productId] : undefined;
+    const selectedProject = selectedProduct ? new Container(getServerContext().container) : undefined;
     const showProductDrawer = selectedProductId === undefined;
     const showLKSDrawer = selectedProductId === LKS_PRODUCT_ID;
-    const showSectionsDrawer = selectedProject !== undefined;
-    const showProjectsDrawer = !selectedProject && selectedProduct !== undefined;
+    const showSectionsDrawer = selectedProduct !== undefined;
+    const { user } = getServerContext();
+    const showMenuSettings = hasPremiumModule() && user.isRootAdmin;
```

Comment:
should this useMemo?

Reply:
I'm not entirely sure it's useful here.  Can `user` change without a reload of the page?